src/chi/components/brand/brand.scss

```diff
@@ -0,0 +1,21 @@
+@import 'mixins';
+@import 'variables';
+
+$brand-sizes: (xs: 0.75rem, sm: 1rem, sm2: 1.25rem, sm3: 1.5rem, md: 2rem, lg: 4rem, xl: 6rem, xxl: 8rem);
+
+.chi {
+  .a-brand {
+    & > img {
+      height: 2rem;
+      width: auto;
```

Comment:
Let's add `display: block;` here to eliminate the whitespace caused by displaying inline. The primary use case for a-brand will be displaying in the header and it will need to be a block element. I'd prefer to style it that way out of the box instead of requiring developers to apply a display utility class.

Reply:
The alternative is applying a `-d--block` utility class on every example in the documentation. @jllr Thoughts?

Reply:
@mattnickles  it makes sense to add display: block to the element itself, instead of requiring developers to apply the display utility class.